docs/en/operations/query-cache.md

```diff
@@ -31,6 +31,10 @@ This reduces maintenance effort and avoids redundancy.
 
 ## Configuration Settings and Usage
 
+:::note
+In ClickHouse Cloud, you must use [query level settings](/en/operations/settings/query-level) to edit query cache settings. Editing [config level settings](/en/operations/configuration-files) is currently not supported.
```

Comment:
Two more questions:

a) The first link (`/en/operations/settings/query-level`) is relative to the top-level directory `docs/`. Does that work? Asking because so far, I have seen in our docs only links relative to the current directory (e.g. `../getting-started/install.md`) or absolute links (e.g. `/docs/en/operations/server-configuration-parameters/settings.md`).

b) The files `query-level.md` and `configuration-files` don't exist in my repo. Typo?

Reply:
a) Our docs framework supports either `/en/` or `/docs/en`. Here's an example of a file that uses `/en/`: https://raw.githubusercontent.com/ClickHouse/clickhouse-docs/main/docs/en/guides/sizing-and-hardware-recommendations.md

b) The files exist but the slugs are redirects, so https://clickhouse.com/docs/en/operations/settings/query-level redirects to `settings-query-level.md`. It's difficult to sustain a 1:1 mapping with markdown paths <-> page URLs because content is constantly being moved around.